weave-js/tailwind.config.cjs

```diff
@@ -4,7 +4,7 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   content: ['./src/**/*.{ts,tsx}'],
-  darkMode: ['class', '[data-mode="dark"]'],
+  darkMode: ['class', '.night-mode'],
```

Comment:
I changed this in core during my night mode updates b/c having two indicators for night mode is unnecessary